src/test/java/com/connexta/search/index/IndexTests.java

```diff
@@ -64,6 +68,8 @@
 
   @Inject private MockMvc mockMvc;
 
+  @Inject private IndexController indexController;
```

Comment:
✏️ Only mockMvc should be used to test the IndexController in this class. Move the tests that need to use indexController directly to another test class.